haystack/nodes/retriever/web.py

```diff
@@ -59,14 +59,17 @@ def __init__(
         search_engine_provider: Union[str, SearchEngine] = "SerperDev",
         top_search_results: Optional[int] = 10,
         top_k: Optional[int] = 5,
-        mode: Literal["snippets", "raw_documents", "preprocessed_documents"] = "snippets",
+        mode: Literal["snippets", "raw_documents", "preprocessed_documents"] = "preprocessed_documents",
```

Comment:
We should keep the default "snippets" and instead change the docstring to `Snippets are the default.`